ui/components/__tests__/__snapshots__/KubeStatusIndicator.test.tsx.snap

```diff
@@ -80,18 +78,11 @@ exports[`KubeStatusIndicator snapshots renders error 1`] = `
   <div
     className="c2 c3 c4"
   >
-    <svg
-      viewBox="0 0 100 100"
-    >
-      <circle
-        cx="50"
-        cy="50"
-        r="45"
-      />
-    </svg>
+    <img
+      src=""
```

Comment:
:thinking: Probably needs to be populated, unless we are using `background-image`?

Reply:
I wonder why nothing is there - maybe because it figures out IconType on load?